src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -107,18 +113,18 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
     `;
   }
 
-  protected shouldUpdate(changedProps) {
+  protected shouldUpdate(changedProps: PropertyValues): boolean {
```

Comment:
This method needs one extra line:

```ts
if (changedProps.has("_config") {
  return true;
}
```

Because we always need to re-render if config has changed.